terraform/aws/analytical-platform-data-production/s3-glue-crawler/glue-catalog.tf

```
@@ -0,0 +1,3 @@
+resource "aws_glue_catalog_database" "alpha_vcms_data_glue_catalog" {
```

```suggestion
resource "aws_glue_catalog_database" "alpha_vcms_data" {
```
Don't need to specify resource type in name